examples/simple/tests/ExampleTest.php

```diff
@@ -16,4 +15,37 @@ public function testItFails()
     {
         $this->assertTrue(false);
     }
+
+    /**
+     * @dataProvider exampleDataProvider
+     */
+    public function testWithDataProvider($actual, $expected)
+    {
+        $this->assertSame($actual, $expected);
+    }
+
+    protected function exampleDataProvider()
```

Comment:
Dataproviders need to be public
```suggestion
    public function exampleDataProvider()
```

Reply:
Oops, missed that. 😅 

Fixed!